pillar/dev/secrets/sentry.sls

```diff
@@ -0,0 +1,5 @@
+secrets:
```

Comment:
we dont need all this nesting i think , 
like how `datadog/init.sls` has `api_key: {{ pillar.get('datadog_api_key') }}`